hugr-py/tests/test_hugr_build.py

```diff
@@ -299,17 +299,18 @@ def test_invalid_recursive_function() -> None:
         f_recursive.set_outputs(f_recursive.input_node[0])
 
 
-def test_higher_order() -> None:
+def test_higher_order(snapshot) -> None:
```

Comment:
issue was originally found as a rendering failure so including that here to make sure rendering now works